src/System.Security.Cryptography.X509Certificates/src/Microsoft/Win32/SafeHandles/PasswordHandle.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+namespace Microsoft.Win32.SafeHandles
+{
+    /// <summary>
+    /// Wrap a string- or SecureString-based object. A null value indicates IntPtr.Zero should be used.
+    /// </summary>
+    internal sealed partial class PasswordHandle : SafeHandle
```

Comment:
I thought it resembled "Secure" too much (as my first naming attempt was SafeSecurePasswordHandle), but since the vast majority of other have the Safe prefix, I changed it to SafePasswordHandle